packages/snap/src/constants/aa-config.ts

```diff
@@ -26,4 +26,13 @@ export const AA_CONFIG = {
     bobaToken: '0x4200000000000000000000000000000000000023',
     bundlerUrl: 'https://bundler.sepolia.boba.network/rpc',
   },
+  // Local
+  [CHAIN_IDS.LOCAL]: {
+    version: '0.6.0',
+    entryPoint: '0x43536f912f87490fa70d0AB6D58f8B9CD250394A',
```

Comment:
these might need to be configured via .env then

Reply:
yeah correct!